Python/Tests/Utilities.Python/MockPackageManager.cs

```diff
@@ -20,6 +20,16 @@ public class MockPackageManager : IPackageManager {
         public event EventHandler InstalledPackagesChanged;
         public event EventHandler IsReadyChanged { add { throw new NotImplementedException(); } remove { } }
 
+        public string ExtensionDisplayName => string.Empty;
+
+        public string IndexDisplayName => string.Empty;
+
+        public string SearchHelpText => string.Empty;
+
+        public string GetInstallCommandDisplayName(string searchQuery) => string.Empty;
```

Comment:
We probably want better values for these, in case we use MockPackageManger instances in UI tests (which I suspect we may, as we fake some Environment Window tests).

Reply:
It's only ever used in a TextBlock that appears in the list of results. It's easy enough to change it, I'll make it match what our pip package manager returns.